extensions/wikia/Email/EmailController.class.php

```diff
@@ -680,4 +665,44 @@ protected function getMessage() {
 			->useDatabase( false )
 			->inLanguage( $this->targetLang );
 	}
+
+	private function trackEmailEvent() {
+		WikiaLogger::instance()->info( 'Submitting email via UserMailer', [
+			'issue' => 'SOC-910',
+			'method' => __METHOD__,
+			'controller' => get_class( $this ),
+			'toAddress' => $this->getToAddress()->toString(),
+			'fromAddress' => $this->getFromAddress()->toString(),
+			'subject' => $this->getSubject(),
+			'category' => static::TRACKING_CATEGORY,
+			'currentUser' => $this->getCurrentUserName(),
+			'targetUser' => $this->targetUser->getName(),
+			'targetLang' => $this->targetLang,
+		] );
+	}
+
+	private function trackEmailError( \Exception $e ) {
```

Comment:
I'm being extra careful with how I get these values here since the error thrown could have been because some of these required values may have been empty.